gufe/tests/test_custom_json.py

```diff
@@ -116,13 +123,125 @@ class TestPathCodec(CustomJSONCodingTest):
     def setup(self):
         self.codec = PATH_CODEC
         self.objs = [
-            pathlib.Path("foo/bar"),
+            pathlib.PosixPath("foo/bar"),
+        ]
+        self.dcts = [
+            {
+                ":is_custom:": True,
+                "__class__": "PosixPath",
+                "__module__": "pathlib",
+                "path": "foo/bar",
+            }
+        ]
+
+
+class TestSettingsCodec(CustomJSONCodingTest):
+    def setup(self):
+        self.codec = SETTINGS_CODEC
+        self.objs = [
+            models.Settings.get_defaults(),
+        ]
+        true = True
+        null = None
+        false = False
```

Comment:
can these be removed / replaced where needed?

Reply:
Removed!